test/Microsoft.Dnx.DesignTimeHost.FunctionalTests/DthStartupTests.cs

```diff
@@ -308,24 +295,24 @@ public void DthNegative_BrokenProjectPathInLockFile_V1(DnxSdk sdk)
         [MemberData(nameof(DnxSdks))]
         public void DthNegative_BrokenProjectPathInLockFile_V2(DnxSdk sdk)
         {
-            var testProject = _fixture.GetTestProjectPath("BrokenProjectPathSample");
+            var projectName = "BrokenProjectPathSample";
+            var project = GetTestProject(sdk, projectName).Restore(sdk, ensureSuccess: false);
 
             using (var disposableDir = new DisposableDir())
             using (var server = DthTestServer.Create(sdk))
             using (var client = server.CreateClient())
             {
-                // copy the project to difference location so that the project path in its lock file is invalid
-                var targetPath = Path.Combine(disposableDir, "BrokenProjectPathSample");
-                Testing.TestUtils.CopyFolder(testProject, targetPath);
+                var movedProjectPath = Path.Combine(disposableDir, projectName);
+                TestUtils.CopyFolder(project.ProjectDirectory, movedProjectPath);
```

Comment:
First time restore it to generate a correct lock file. Then move it to another place but do not move its dependency so that the relative path in lock file is invalid.